root/etc/e-smith/templates/etc/opt/rh/rh-php72/php-fpm.d/000-nextcloud.conf/10base

```diff
@@ -0,0 +1,62 @@
+{
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro() || die("Can't open config db");
+
+    $OUT .= q(
+;
+;  nextcloud php-fpm configuration
+;
+
+[netserver-nextcloud]
+listen = /var/run/rh-php72-php-fpm/nethserver-nextcloud-php72.sock
+
+;Logs
+php_admin_value[error_log] = /var/opt/rh/rh-php72/log/php-fpm/error-nextcloud.log
```

Comment:
Make sure to mention this change in the README

Reply:
done